### PR TITLE
Feature/delete-post & Feature/delete-comment

### PR DESCRIPTION
**Pull request by Boyce-Lee**
# what I done
## 3. add route map in route.go for deleting post and comment
```go
// in route.go
func registerCommentRoutes(app *iris.Application) {
	// redundant API "/comments" for "/posts/{postid:int min(1)}/comments"
	commentRoutes := app.Party("/post/{postid:int min(1)}").Party("/comments")
	commentRoutes.Use(middlewares.CheckLoginStatus)

	// add any subpath below
	// commentRoutes.Get("/", service.GetComments)
	commentRoutes.Delete("/{commentid:int min(1)}", service.DeleteComment)
}
```

### CODE REVIEW

File: vendor/model/user_upvote_post.go [hunk 1]
```diff
@@ -7,7 +7,7 @@ import (
 
 // UpvotePostByUser ..
 func UpvotePostByUser(userid, postid int64) (int64, error) {
-	return database.Orm.Table("UserUpvotePost").Insert(
+	return database.Orm.Table("user_upvot_post").Insert(
```
Comment (MarshallW906) [inline vendor/model/user_upvote_post.go]: `user_upvote_post`? I mean, a missing `e`

**Review by MarshallW906 — CHANGES_REQUESTED** [review]
Everything's OK but a tiny spelling error. 👍

File: vendor/model/user_upvote_post.go (hunk 1 above)
Comment (Boyce-Lee) [reply vendor/model/user_upvote_post.go]: i has fix the error

**Review by alexandrali3 — DISMISSED** [review]
plz separate the fixing part of route.go from this branch.

**Review by MarshallW906 — APPROVED** [review]

**Pull request merged**